code/nrf-connect/prstlib/src/sensors.c

```diff
@@ -19,7 +19,7 @@ int prst_sensors_read_all(prst_sensors_t *sensors) {
   LOG_DBG("Soil: %.0f %% (%d mV)", 100 * sensors->soil.percentage,
```

Comment:
Tks @seime, nice catch.

I wrote https://github.com/rbaron/b-parasite/issues/206#issuecomment-2146762888 explaining that for now we should remove the `millivolt` logging from this line. Let me know if you can do this in the PR as well, otherwise I will send another one.

Tks!